### PR TITLE
[exomiser] Support starting at most 10 exomisers in parallel

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

File: rpm/src/main/resources/exomiser_cron.py [hunk 1]
```diff
@@ -314,40 +316,53 @@ def run_exomiser(record_id, new_data, settings):
...
-                if not changed:
-                    logging.info("{0} hasn't changed".format(record_id))
-                    continue
-
-                # Patient record has changed, (re-)run Exomiser
-                run_exomiser(record_id, new_data, settings)
-        except RecordLockedException:
-            logging.error('Skipping locked record: {0}'.format(record_id))
+    exomiser_results = []
+    with contextlib.closing(Pool(processes=10)) as pool:
```
Comment (sdumitriu) [inline rpm/src/main/resources/exomiser_cron.py]: This should be a parameter defined above with the rest of the parameters.

**Comment by buske** [comment]
"exomizer-scrypt" 😢

**Comment by veronikaslc** [comment]
@buske sorry, do you want me to rename branch to a better version?

**Comment by buske** [comment]
Just FYI, it's "exomiser" and "script" :)

**Comment by veronikaslc** [comment]
Coled in favour of https://github.com/phenotips/phenomecentral.org/pull/87

**Pull request closed without merging**